cpu/atmega128rfa1/include/atmega_pcint.h

```diff
@@ -0,0 +1,15 @@
+#ifndef ATMEGA_PCINT_H
+#define ATMEGA_PCINT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define ATMEGA_PCINT_MAP_PCINT0 GPIO_PIN(PORT_B, 0), GPIO_PIN(PORT_B, 1), GPIO_PIN(PORT_B, 2), GPIO_PIN(PORT_B, 3), GPIO_PIN(PORT_B, 4), GPIO_PIN(PORT_B, 5), GPIO_PIN(PORT_B, 6), GPIO_PIN(PORT_B, 7)
+#define ATMEGA_PCINT_MAP_PCINT1 GPIO_PIN(PORT_E, 0), GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF, GPIO_UNDEF
```

Comment:
I didn't thought this through, but maybe PCINT1 can be treated like an EXTINT if there is only one pin triggering the interrupt? If this would be possible to do with little effort and code spend on it, this should be more efficient.

Reply:
TBH, I just copied that from `cpu/atmega256rfr2/include/atmega_pcint.h`.

Looking at the data sheet confirms that `PE0` is the only `PCINT` on this port.
Is there already a define for what you suggest, or would that require changes to `atmega_common/periph/gpio.c`? If so, I'd rather do that in a later PR.

Reply:
Looking at it, it seems like `CPU_ATMEGA_EXT_INTS` is then wrong for both - it should be the same pins as the ones defined in `ATMEGA_PCINT_MAP_PCINTx`, right?

Reply:
No, those are two different types of interrupts. The EXT_INTS have one interrupt vector per pin, the PCINT have one interrupt vector per group of up to 8 pins. So for PCINT the ISR has to figure out which pin triggered the interrupt, if that info is needed. (For compliance with the RIOT API is is always needed.)